src/app_service/service/token/service.nim

```diff
@@ -304,4 +317,4 @@ QtObject:
       timeInterval: timeInterval
     )
     self.threadpool.start(arg)
-    return
\ No newline at end of file
+    return
```

Comment:
(still a NIM newb): I wonder what a `return` like this as the last statement is doing

Reply:
Yeah I know it showed up merely because of the EOF marker change